christmas/noscript.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CSS Christmas</title>
+    <link rel="icon" href="../favicon.ico">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="envelope open">
+        <div class="front">
+            <div class="mailme">
+            <p>A special message from CSS</p>
+            </div>
+        </div>
+        <div class="back">
+            <div class="letter">
+              <img src="chrimsas.jpg" alt="" style="height: 75%; border-radius: 5px; position: absolute; left: 5px; top: 45px;">
+              <div style="position: absolute; right: 10px; top: 50px; width: 46%; text-align: center;">
```

Comment:
Inline CSS. Extract to CSS file?

Also on mobile: 
![image](https://user-images.githubusercontent.com/20641536/102879565-d7dbc780-4441-11eb-8f89-9ee5784533bf.png)

Also doesn't properly resize on normal browser window, making text hard to read.

Reply:
noscript.html has no scaling as the approach relies on zoom and transform: scale (for Firefox only).